.changeset/little-dingos-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
you don't have to add changesets to the internal services (i think? @kamilkisiela ?)

Reply:
yes, no need